### ISSUE TITLE
Amazing log

### ISSUE THREAD

**Issue by OxFEE1DEAD** [rs/zerolog#585]
This log looks really great, but it is implemented with slog

![image](https://github.com/rs/zerolog/assets/58794405/7e74f304-bfac-4e5e-8fb1-6996c5791688)

Does somebody know how we can implement it with zerolog. Is it possible with zerolog.ConsoleWriter ?

**Comment by mitar** [comment]
Yes, `ConsoleWriter` can be configured to format logs in this way. See its `FormatTimestamp`, `FormatLevel`, and `FormatFieldValue` fields. This should be trivial.
